### PR TITLE
Update README to add error output

### PR DESCRIPTION
**Pull request by weikinhuang**

### CODE REVIEW

**Review by kaicataldo — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "test:dev": "npm run karma -- --reporters=mocha,coverage",
     "eslint": "eslint src/**/*.js test/**/*.js",
     "karma": "cross-env NODE_ENV=test karma start",
-    "webpack": "webpack --progress --color",
+    "webpack": "cross-env NODE_ENV=production webpack --progress --color",
```
Comment (nemtsov) [inline package.json]: Just curious: why lock it to production?

Comment (weikinhuang) [reply package.json]: no reason it doesn't really do anything

Comment (nemtsov) [reply package.json]: Ha! Why do it then?

Comment (weikinhuang) [reply package.json]: Fine, I'll remove it

**Review by nemtsov — APPROVED** [review]

**Pull request merged**